internal/lsp/token_encoder_test.go

```diff
@@ -80,6 +80,59 @@ func TestTokenEncoder_singleLineTokens(t *testing.T) {
 	}
 }
 
+func TestTokenEncoder_unknownTokenType(t *testing.T) {
```

Comment:
I'm unsure if this test case covers the bug in https://github.com/hashicorp/terraform-ls/issues/763. If I undo the changes in `internal/lsp/token_encoder.go`, it still passes. Should we extend the test?

Reply:
Test modified, thanks for the catch!